index.html

```diff
@@ -282,12 +264,9 @@ <h2>Contact me</h2>
               required
             ></textarea>
```

Comment:
- [ ] Same here, Kindly ensure that the text area input field accepts up to 500 chars,

Reply:
I set maxlength="500" in the "text-area" input.